coordinator/internal/logic/provertask/batch_prover_task.go

```diff
@@ -61,13 +69,46 @@ func (bp *BatchProverTask) Assign(ctx *gin.Context, getTaskParameter *coordinato
 		return nil, fmt.Errorf("check prover task parameter failed, error:%w", err)
 	}
 
+	var hardForkNumber uint64
+	if getTaskParameter.HardForkName != "" {
+		var exist bool
+		hardForkNumber, exist = bp.nameForkMap[getTaskParameter.HardForkName]
+		if !exist {
+			log.Debug("hard fork prover get empty batch because of the hard fork name don't exist", "height",
```

Comment:
Later we can add a metric to track this. And to track number of task requests by different fork.

Reply:
fixed [9399974](https://github.com/scroll-tech/scroll/pull/1182/commits/93999746c9612d06f0c99657255d385d5d6183c9)